whois-commons/pom.xml

```diff
@@ -134,6 +134,11 @@
             <groupId>org.elasticsearch.client</groupId>
             <artifactId>elasticsearch-rest-high-level-client</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Why we need this dependency ?

Reply:
import org.eclipse.jetty.util.StringUtil;
We are using this in "WhoisHazelcastBlockedIps"

Reply:
No we should use StringUtils from lang package

Reply:
I removed anyways this

Reply:
whoisDosFilter and blocklistfilter are in same module so there should not be a need to add dependency but thanks for removing it

Reply:
Ahh got it, I don't think this needs to be in common package. As it will not be used everywhere , do you think it makes more sense to move to some other module? May be whois-api?